extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/config/KubernetesConfig.java

```diff
@@ -0,0 +1,421 @@
+package io.quarkus.kubernetes.deployment.config;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import io.dekorate.kubernetes.annotation.ImagePullPolicy;
+import io.dekorate.kubernetes.annotation.ServiceType;
+import io.quarkus.kubernetes.deployment.DeploymentTarget;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
+public class KubernetesConfig implements PlatformConfiguration {
+
+    /**
+     * The group of the application. This value will be use as: - docker image repo
+     * - labeling resources
+     */
+    @ConfigItem
+    Optional<String> group;
+
+    /**
+     * The name of the application. This value will be used for naming Kubernetes
+     * resources like: - Deployment - Service and so on ... If no value is specified
+     * it will attempt to determine the name using the following rules: If its a
+     * maven/gradle project use the artifact id. Else if its a bazel project use the
```

Comment:
I don't think we want to keep the Bazel reference here since it doesn't apply to the Quarkus case